log-collector-script/linux/eks-ssm-content.json

```diff
@@ -34,7 +34,7 @@
                "/usr/local/bin/aws --version",
                "fi",
                "echo \"Pushing to S3\"",
-               "/usr/local/bin/aws s3 cp /var/log/eks_i* s3://{{bucketName}}",
+               "/usr/local/bin/aws s3 cp $(ls -1 /var/log/eks_i* | tail -1) s3://{{bucketName}} && rm /var/log/eks_i*",
```

Comment:
Thanks for contributing. Maybe we should separate these statements? More specifically, I think we should move `rm /var/log/eks_i*` to run before we run the log collector so that we clean any previous runs first. This will remove the need to do any `ls` and `tail` massaging. I'm trying to think if there's a reason we'd want to keep the bundle on the instance after the run for troubleshooting or something, but probably not? If not, we could just run it before (for any missed ones) and after, if we want to reclaim the disk space.

Reply:
As suggested, I placed the cleanup on top.